docs/UserGuide.md

```diff
@@ -187,18 +188,18 @@ Examples:
 
 ### Deleting a customer : `deletecustomer`
 
-Deletes the specified customer from the address book.
+Deletes the specified customer contact and order list from the address book.
```

Comment:
customer's contact and order list?

Reply:
But I not sure can @kaixuan477 help please

Reply:
```suggestion
Deletes the specified customer contact and their corresponding orders from the address book.
```